src/framework/mlt_image.c

```diff
@@ -581,3 +581,20 @@ void mlt_image_format_planes( mlt_image_format format, int width, int height, vo
 		strides[3] = 0;
 	};
 }
+
+
+
+/** Check if the alpha channel of an rgba image is opaque
+ *
+ * \public \memberof mlt_image_s
+ * \param image the image buffer
+ * \param width width of the image in pixels
+ * \param height height of the image in pixels
+ */
+int mlt_image_rgba_opaque(uint8_t *image, int width, int height)
```

Comment:
This function should operate on an mlt_image object. Here is a proposed refactor (NOT TESTED)!
https://github.com/bmatherly/mlt/commit/a408493198ed517c1d42e8536d4d0e0c287576f0

Reply:
I disagree. I do not want to require constructing a mlt_image object just to use this function. Look at where it came from and how it’s being used.

Reply:
Hmm. OK. I guess I do not feel too strongly about it. But if we do not write it to take an mlt_image struct, that means we will have to deprecate this new function eventually when the code is changed to use mlt_image and I will probably be the one to rewrite the new function. Did you view my proposed change? It is only 2 extra lines of code to create the mlt_image object and make the function future proof.

Also, this function would be the only function in mlt_image.h that operates on a raw buffer instead of an mlt_image struct. But I do not think of another better place to put it.

Reply:
I did consider the mlt_image-based approach before suggesting `mlt_image_rgba_opaque` to avoid a future name conflict. Then, there would not not necessarily be a need to break API. I am going to accept this PR as-is for JB's sake. Please refactor it afterward to your `mlt_image_is_opaque` and drop `mlt_image_rgba_opaque` before release, which will be by end of year at the latest but possibly as soon as end of October.

Reply:
> I am going to accept this PR as-is for JB's sake. Please refactor it afterward to your mlt_image_is_opaque and drop mlt_image_rgba_opaque before release, which will be by end of year at the latest but possibly as soon as end of October.

Agreed. Let's get this one merged.